internal/pkg/testhelper/vt_windows.go

```diff
@@ -0,0 +1,15 @@
+// +build windows
+
+package testhelper
+
+import (
+	"testing"
+
+	"github.com/ActiveState/vt10x"
+	"github.com/Netflix/go-expect"
+)
+
+func NewVirtualTerminal(t *testing.T, opts ...expect.ConsoleOpt) (*expect.Console, *vt10x.State, error) {
+	t.Helper()
+	t.Skipf(`virtual terminal in not supported in Windows tests`)
```

Comment:
Nepodarilo sa mi rozbehat interaktivne testy na `Windows`, .. takze tu pridavam `skip` a implementaciu vraciam spat, s pouzitim `pty` na Linux/MacOs to ide v pohode.

Problem je, ze kniznica `expect` si Windowsom velmi nerozumie. A pouzivat nejaku specialnu kniznicu pre Windows by bolo vela prace.